.github/workflows/release-porting-guide.yml

```
@@ -0,0 +1,103 @@
+---
+name: Ansible porting guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-build-data-branch:
+        description: >-
+          Release Branch name from the Ansible Build data PR.
+          (e.g. `refs/pull/1/merge`)
+        required: true
+      ansible-version:
+        description: >-
+          Exact release version. For exmaple, 12.1.0
+        required: true
+
+jobs:
+  upload-porting-guide:
+    name: Extract the porting guide
+    runs-on: ubuntu-latest
+    env:
+      GIT_BRANCH: "release/porting-guide-${{ inputs.ansible-version }}"
+      CI_COMMIT_MESSAGE: >-
+        Add the Ansible community "${{ inputs.ansible-version }}" porting guide
```

I don't think the quotes look good in the commit message.
```suggestion
        Add the Ansible community ${{ inputs.ansible-version }} porting guide
```